src/reducers/membersNew.ts

```diff
@@ -207,21 +208,21 @@ function applyOperation(
 
     switch (operation.op_code) {
       case OperationType.REQUEST_INVITE: {
-        const { full_name, to_uid } = operation.data;
+        const { full_name, to_uid, username } = operation.data;
```

Comment:
Still not a huge fan of processing every single op on client in order to build state ...